pysheaf/pysheaf.py

```diff
@@ -993,25 +1034,23 @@ def optimize_SLSQP(self, assignment, activeCells=None, testSupport=None, tol=1e-
             Tech. Rep. DFVLR-FB 88-28, DLR German Aerospace Center, 
             Institute for Flight Mechanics, Koln, Germany.
         """
-    
         initial_guess, bounds = self.serializeAssignment(assignment,activeCells)
-        res=scipy.optimize.minimize( fun = lambda sec: self.assignmentMetric(assignment,self.deserializeAssignment(sec,activeCells,assignment), testSupport=testSupport),
+        res=scipy.optimize.minimize( fun = lambda sec: ((self.assignmentMetric(assignment,self.deserializeAssignment(sec,activeCells,assignment), testSupport=testSupport))),
                                     x0 = initial_guess,
                                     method = 'SLSQP', 
                                     bounds = bounds,
                                     constraints = ({'type' : 'eq',
                                                     'fun' : lambda asg: self.consistencyRadius(self.deserializeAssignment(asg,activeCells,assignment),testSupport=testSupport)}), 
                                     tol = tol, 
-                                    options = {'maxiter' : int(100)})
+                                    options = {'maxiter' : int(100), 'disp' : True})
```

Comment:
Probably will not want to retain this for production?

Reply:
I was thinking it was good to leave, because it automatically displays the problems with SLSQP for most problems.